Handle IPSL-CM6 (#1153)
ESMValGroup/ESMValCore: doc/quickstart/configure.rst

`esmvalcore/config-developer.yml
<https://github.com/ESMValGroup/ESMValCore/blob/main/esmvalcore/config-developer.yml>`_.
This configuration file describes the file system structure and CMOR tables for several
key projects (CMIP6, CMIP5, obs4mips, OBS6, OBS) on several key machines (e.g.
BADC, CP4CDS, DKRZ, ETHZ, SMHI, BSC), and for native output data for some

> models (IPSL, ... see :ref:`configure_native_models`) .

Reviewer: ```suggestion
models (IPSL, ... see :ref:`configure_native_models`).
```

Author: Done

  to get the name of the file containing the ``mip`` table.
  Defaults to the value provided in ``cmor_type``.

.. _configure_native_models:
  
Configuring native models and observation data sets
----------------------------------------------------

> ESMValTool can take full advantage of the ability to configure

Reviewer: ```suggestion
ESMValCore can take full advantage of the ability to configure
```

Author: Done

ESMValCore for handling native model output formats and specific
observation data sets without preliminary reformating. You can choose
to host this new data source either under a dedicated project or under
project ``native6``; when choosing the latter, such a configuration
involves the following steps :

  - allowing for ESMValTool to locate the data files :

    - entry ``native6`` of ``config-developer.yml`` should be
      complemented with sub-entries for ``input_dir`` and ``input_file``
      that goes under a new key representing the

>       data organization (such as ``NEW_MODEL``), and these sub-entries can

Reviewer: ```suggestion
      data organization (such as ``INSTITUTE``), and these sub-entries can
```
these names are usually institute names, not model names, because they refer to the data storage organization used at a particular institute.

      use an arbitrary list of ``{placeholders}``. Example :

      .. code-block:: yaml

	native6:
  	  ...
	  input_dir:
             default: 'Tier{tier}/{dataset}/{latestversion}/{frequency}/{short_name}'
             NEW_MODEL: '{model}/{exp}/{simulation}/{version}/{type}'
          input_file:
            default: '*.nc'
            NEW_MODEL: '{simulation}_*.nc'
          ...

    - if necessary, provide a so-called ``extra facets file`` which
      allows to cope e.g. with variable naming issues for finding
      files. See `Extra_Facets`_  and here :download:`an example of
      such a file for IPSL-CM6
      <../../esmvalcore/_config/extra_facets/native6-ipsl-cm6-mappings.yml>`.

>   - ensuring that ESMValTool get the right metadata and data out of

Reviewer: ```suggestion
  - ensuring that ESMValCore gets the right metadata and data out of
```

Author: Done

>     your data files : this is described at :ref:`fixing_data`

Reviewer: ```suggestion
    your data files: this is described in :ref:`fixing_data`
```

Author: Fixed
